src/script.js

```diff
@@ -0,0 +1,17 @@
+var state = localStorage.getItem('toggle') === 'true' ? true : false
```

Comment:
Please use a more descriptive name like "darkMode" or "style" and then set it to "light" or "dark".

Reply:
ok i tried to improve on this! please see!